### DISCUSSION TITLE
[bug] Linking your own game from a previous LD in a comment causes a warning against self-advertising

### DISCUSSION THREAD

**Discussion by HolyBlackCat** [ludumdare/ludumdare#2117] [General]
I found a game (on this LD) that was similar to something I made for the *previous* LD, and wanted to link it in the comments.

When I tried to do it, I got an automatic warning, telling me to avoid excessive advertising.

I feel this should apply only when linking your entry for the *current* LD, not the previous ones.

**Comment by mikekasprzak** [comment]
It's still a reasonable request not to spam, but I see your point. I can't remember the wording or urgency used, but that could certainly be looked at.

Today the website only runs Ludum Dare events, but we will eventually run and allow other events. These events may overlap, and it would be sensible to discourage events from spamming each-other.
